translation from c00/30/16 to 20 (#66)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/17/2/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//
ありがとうございます、本当に助かります。=Thank you, you really helped me out.

> 本当はまだまだお礼の言葉を述べたいのですが……すみません、失礼します。=I would really like to thank you some more, but... I'm sorry, I need to go.

Reviewer: ```suggestion
本当はまだまだお礼の言葉を述べたいのですが……すみません、失礼します。=I would really like to thank you some more, but...  I need to go, sorry.
```

一層あなたのことを好きになりました。改めてお礼を言わせてください、ありがとうございます。=I'm falling for you even more. Thank you again for your kindness.
このご恩は忘れません。ありがとうございます。=I won't forget this favor. Thank you very much.
あなたのご厚意を無駄にしないためにも、お手洗いへ急ぎます。本当にありがとうございました。=I will hurry to the bathroom so that your kindness doesn't go to waste. Thank you for your help.

> お手数をおかけしました。同じ失態をしないように心に留めておきます。=I appologise for the inconvenience. I'll try not to make the same blunder again.

Reviewer: `blunder` -> `mistake`

……ありがとうございます。=...Thank you.

> お陰で大事に至らずに済みました。改めてお礼申し上げます。=Thanks to you, it didn't become a serious problem. Thank you once again.

Reviewer: Trying to be less redundant with the thanks, but also keep that she feels indebted
```suggestion
お陰で大事に至らずに済みました。改めてお礼申し上げます。=Thanks to you, it didn't become a serious problem. I owe you one, again.
```

```suggestion
お陰で大事に至らずに済みました。改めてお礼申し上げます。=Because of you, it didn't become a serious problem. Thank you, again.
```

> 優しい一面もあるのですね。少しだけ見直しました。=So there's a gentle side to you, isn't there? I've changed my opinion a little.

Reviewer: More natural sounding?
```suggestion
優しい一面もあるのですね。少しだけ見直しました。=So there's a gentle side to you, isn't there? Maybe I was wrong about you.
```

> ……襲わないのですか？　いえ、がっかりしてませんよ。=...You're not going to take advantage of me? No, I'm not dissapointed.

Reviewer: bit more wistful

```suggestion
……襲わないのですか？　いえ、がっかりしてませんよ。=...You're not going to take advantage of me? Oh, no, I'm not disappointed.
```

ありがとうございました。不快な思いをさせてはいけないので、私はこれで失礼します。=Thank you very much. I'm getting really uncomfortable, so please excuse me.

> 誠にありがとうございました。……もしかして、アブノーマルなセックスは好みではなかったのでしょうか？=Thank you for your help. ...Could it be, you're not into abonormal sex?

Reviewer: ```suggestion
誠にありがとうございました。……もしかして、アブノーマルなセックスは好みではなかったのでしょうか？=Thank you for your help. ...Maybe you're not into kinky sex?
```

Author: KINKY, that was the term! god, I knew there was a term for this that i couldn't recall for the life of me! :D  thank you :D
